app/page.jsx

```diff
@@ -1,18 +1,63 @@
 'use client';
 
-import LoginButton from './(button)/LoginButton';
-import useAuth from '../src/hooks/useAuth';
+import Image from 'next/image';
+import LoginButton from 'app/(button)/LoginButton';
+import PeopleListening from 'app/(illustrations)/PeopleListening';
+import Phone from 'app/(illustrations)/Phone';
+import PeoplePlaying from 'app/(illustrations)/PeoplePlaying';
+import PeopleDancing from 'app/(illustrations)/PeopleDancing';
+import useAuth from '@/hooks/useAuth';
+import logo from '../public/images/logo.png';
 
 export default function Home() {
   useAuth();
 
   return (
-    <div className="text-green-500">
-      I am Home (AKA Now Playing)
-      <br />
-      <br />
-      <div>
-        <LoginButton />
+    <div className="bg-landing min-h-screen font-primary">
```

Comment:
Try this out, but I feel like for some reason the default settings on font are slapping Poppins out of the water here...
```suggestion
    <div className="bg-landing min-h-screen">
```